src/krb5_plugin/sssd_krb5_locator_plugin.c

```diff
@@ -569,22 +570,32 @@ krb5_error_code sssd_krb5_locator_lookup(void *private_data,
             return KRB5_PLUGIN_NO_HANDLE;
```

Comment:
Since there can be multiple addresses and names in the kdcinfo file and actually DNS lookups might be done now I think it might be better to have the possibility not fail completely but just try the next entry.

Just error out to avoid further delays with upcoming DNS lookups (which have a chance to fail as well) is imo a sensible default.

But I wonder what you think about adding another environment variable like e.g. SSSD_KRB5_LOCATOR_IGNORE_DNS_FAILURES to tell the plugin to ignore the error and continue with the next entry?